controllers/idler/idler_controller.go

```diff
@@ -197,10 +212,18 @@ func (r *Reconciler) createNotification(logger logr.Logger, idler *toolchainv1al
 			// no email found, thus no email sent
 			return fmt.Errorf("no email found for the user in MURs")
 		}
-
+		if len(appName) == 0 {
+			appName = ""
+		}
+		if len(appType) == 0 {
+			appType = ""
+		}
```

Comment:
wondering if we really need this - if the len of the string is 0 isn't the string already blank or am I missing something ?

Reply:
i can remove this ,its not required, it was written previously so that it doesn't break something else but. i handled that scenario differently

Reply:
Let's delete this :) This code doesn't really do anything.